cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -133,24 +99,12 @@ def _cleanup():
                                                              'Details').VIEW)
     view.wait_displayed()
     request_details_list = view.migration_request_details_list
-    vm = request_details_list.read()[0]
-
-    def _get_plan_status():
-        clock_reading1 = request_details_list.get_clock(vm)
-        time.sleep(1)  # wait 1 sec to see if clock is ticking
-        logger.info("For vm %s, current message is %s", vm,
-            request_details_list.get_message_text(vm))
-        logger.info("For vm %s, current progress description is %s", vm,
-            request_details_list.get_progress_description(vm))
-        clock_reading2 = request_details_list.get_clock(vm)
-        logger.info("clock_reading1: %s, clock_reading2:%s", clock_reading1, clock_reading2)
-        logger.info("For vm %s, is currently in progress: %s", vm,
-          request_details_list.is_in_progress(vm))
-        return not(request_details_list.is_in_progress(vm) and (clock_reading1 < clock_reading2))
-
-    wait_for(func=_get_plan_status, message="migration plan is in progress, be patient please",
+    vms = request_details_list.read()
+
+    wait_for(func=view.get_plan_status, message="migration plan is in progress, be patient please",
      delay=5, num_sec=3600)
-    assert (request_details_list.is_successful(vm) and not request_details_list.is_errored(vm))
+    assert (request_details_list.is_successful(vms[0]) and
```

Comment:
I think there's a bit of an assumption here that could cause some exceptions - and really its a hidden test assertion.

Add a check that there is a vm in `vms` list, before indexing it, when it may be empty.